src/formatDate.js

```diff
@@ -7,8 +7,39 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // Розділяємо дату на частини за допомогою роздільника у форматі fromFormat
+  const separator = fromFormat.pop(); // Останній елемент - це роздільник
```

Comment:
Modifying the `fromFormat` array directly by using `pop()` can lead to unintended side effects if `fromFormat` is used elsewhere after calling `formatDate`. It's better to access the separator without altering the original array.